### DISCUSSION TITLE
How to move the note to the subfolder of a folder

### DISCUSSION THREAD

**Discussion by erfanbashar1q** [RafaelGB/obsidian-db-folder#566] [Q&A]
Hello

how can I move a note to the subfolder of a folder, what i mean is i want to move one note to the folder/subfolder based on the feature of the plugin that is "Moving Notes Based on Metadata".
i made a select column, then for a note that i want to move to the folder/subfolder use this : folder/subfolder. but, in the vault it makes a folder with name of folder/subfolder, not make a folder, then make a subfolder for the former folder and then move the note into it.

could you please tell me how should i solve this? thanks

**Comment by imed-ghomari** [comment]
Could you post a screen recording with the result you want to achieve? This is a bit confusing to understand, I made sure to simplify as much as possible when making that part of the documentation, so you should be able to understand what you can do from there. 

Otherwise, explain with a video, and I'll try to see if it can be done.

**Reply by erfanbashar1q** [reply]
https://user-images.githubusercontent.com/116307599/199120431-583ecc96-b0dc-4928-8acc-4accfe4bcc17.mp4

**Reply by imed-ghomari** [reply]
Ah I see, the problem is how you named the option in the select property `z1/z2`, the plugin doesn't recognize this format.
If you want to nest the note two folders deep, the only way is to create two different select columns :
- one for the first level (column for z1) 
- and another for the second level (column for z2).

If you don't like this solution, you can always make a feature request.
